pkgs/applications/office/zanshin/default.nix

```diff
@@ -1,10 +1,18 @@
-{
```

Comment:
To the commit message: Shouldn't be "broken after the 20.08.2 upgrade"? It would be also nicer to comment near the `broken = true` why is it broken.

Reply:
No, it was broken before. We just didn't tag it as such.